front/src/pages/categories/[categoryId].jsx

```diff
@@ -7,35 +7,31 @@ const category = {
 }
 
 const products = [
```

Comment:
Isn't this dead code since you're querying our backend ?

Reply:
I will make the backend call